packages/contracts/contracts/L1/interfaces/IValidatorManager.sol

```diff
@@ -21,8 +21,7 @@ interface IValidatorManager {
      * | Status            | initiated | started | MIN_REGISTER_AMOUNT | MIN_START_AMOUNT |
      * +-------------------+-----------+---------+---------------------+------------------+
      * | NONE              | X         | X       | X                   | X                |
-     * | INACTIVE          | O         | X       | X                   | O/X              |
-     * | IN_JAIL           | O         | O/X     | O/X                 | O/X              |
+     * | INACTIVE          | O         | O/X     | X                   | O/X              |
      * | ACTIVE            | O         | X       | O                   | X                |
      * | CAN_START         | O         | X       | O                   | O                |
      * | STARTED           | O         | O       | O                   | X                |
```

Comment:
`ACTIVE` looks good to me, but `ELIGIBLE` looks quite ambiguous to me. So how about like below?

- `INACTIVE` -> `UNSTAKED`
- `ACTIVE` -> `REGISTERED`
- `CAN_START` -> `READY`
- `STARTED` -> `INACTIVE`
- `CAN_SUBMIT_OUTPUT` -> `ACTIVE`

Reply:
Or `NOT_REGISTERED`?

Reply:
Do you mean it for `UNSTAKED`?

Reply:
cc. @sm-stack validator client also needs to apply this change

Reply:
I think `NOT_REGISTERED` means that the validator should re-register, but the validator can only delegate more than `MIN_REGISTER_AMOUNT`. So how about `EXITED`? Because if the validator undelegates the asset left less than `MIN_REGISTER_AMOUNT`, it means the validator is not willing to act as a validator.

Reply:
It looks good to me.